hail/src/main/scala/is/hail/methods/IBD.scala

```diff
@@ -23,15 +23,16 @@ object IBDInfo {
 
   val pType =
     PStruct(("Z0", PFloat64()), ("Z1", PFloat64()), ("Z2", PFloat64()), ("PI_HAT", PFloat64()))
-
+  
+  //FIXME: remove fromRegionValue?
```

Comment:
can replace RegionValue occurrences with Long and delete Region args. remove the FIXME though